.editorconfig

```diff
@@ -0,0 +1,14 @@
+; Top-most http://editorconfig.org/ file
+root = true
+charset = utf-8
+
+; Unix-style newlines
+[*]
+end_of_line = LF
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+; 4-column tab indentation
+[*.yml]
+indent_style = space
```

Comment:
Space?

Reply:
GitHub Actions workflows normally use spaces for indentation. We can probably change that if you want.

Reply:
Not strictly necessary (although I prefer tabs). We should be consistent with the comment above though.

Reply:
![image](https://github.com/OpenRA/Eluant/assets/7137365/c2d2aa72-b4d5-491e-9abf-84ff6a37c368)
So I'll just remove the misleading comment from the `.editorconfig`.